service.py

```diff
@@ -3,18 +3,22 @@
 import os
 import sys
 import xbmc
-import urllib
-import urllib2
+if sys.version_info.major == 3:
+    import urllib.request, urllib.parse, urllib.error
+    import html.parser
+else:
+    import urllib
+    import urllib2
+    import HTMLParser
 import xbmcvfs
 import xbmcaddon
 import xbmcgui
 import xbmcplugin
 import uuid
 import unicodedata
 import re
-import string
+#import string
```

Comment:
Remove this line if not needed.

Reply:
I replaced all explicit string calls like string.lower(found_title) to found_title.lower() which works both in python 2 and 3. Importing the string module isn't required any longer as it's not being used any longer.

Reply:
I'll get rid of it in the next commit :)

Reply:
Get rid of it and I'll merge :)

Reply:
done